CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 
 ## Master
 
-#### User Interface
+#### User interface
```

Comment:
Why are we Caps-casing everything but "User interface"?

Reply:
Everything should be in sentence case. “Core Navigation” is a framework name, so it just looks like title case.

Reply:
![Aha! Totally makes sense.](https://i.imgur.com/5qghkYY.gif)